scripts/verify_ai_config_flow.py

```
@@ -0,0 +1,156 @@
+  5. POST 塞入密文格式 key → 还原明文，不污染缓存
+  6. POST 掩码（含 ****）→ 不覆盖原 key
+  7. 密文损坏 / 换机器 → key 安全清空，不崩溃、不用密文
+  8. 空 key（ollama/localhost 场景）→ 不加密
+"""
+import json
+import os
+import sys
+
+sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
+# 复用协议验证脚本里的加解密实现（与 C++ 一一对应）
+from verify_secure_store import encrypt_cpp, decrypt_cpp, K_MAGIC, b64_decode  # noqa: E402
```

Flake8 (7.3.0)

[error] 'verify_secure_store.b64_decode' imported but unused

(F401)